op-chain-ops/cmd/celo-dbmigrate/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime/debug"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/mattn/go-isatty"
+	"golang.org/x/exp/slog"
+)
+
+// How to run:
+// 		go run ./op-chain-ops/cmd/celo-dbmigrate -oldDB /path/to/oldDB -newDB /path/to/newDB [-batchSize 1000] [-verbosity 3] [-memoryLimit 7500] [-clear-all] [-clear-nonAncients]
+//
+// This script will migrate block data from the old database to the new database
+// You can set the log level using the -verbosity flag
+// The number of ancient records to migrate in one batch can be set using the -batchSize flag
+// The default batch size is 1000
+// You can set a memory limit in MB using the -memoryLimit flag. Defaults to 7500 MB. Make sure to set a limit that is less than your machine's available memory.
+// Use -clear-all to start with a fresh new database
+// Use -clear-nonAncients to keep migrated ancients, but not non-ancients
```

Comment:
I would scrap this and move the extra info into the flag descriptions (such as for the memory limit). Also there doesn't seem to be a verbosity flag.

Reply:
Also the state migration script is using `github.com/urfave/cli/v2` which produces nicer looking output and lets you provide a description of the program overall, so when we merge the two scripts I think we should use that.